src/main/java/uk/gov/pay/adminusers/model/Service.java

```diff
@@ -19,6 +19,8 @@
 @JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
 public class Service {
 
+    public static final String FIELD_NAME = "name";
```

Comment:
is there a reason why `FIELD_NAME` is in this class when it's only used by `ServiceResource`?

Reply:
When I moved it, it was used in other places too but those usages have been removed now, so I’ll move the constant to `ServiceResource`.